frontend/packages/topology/console-extensions.json

```diff
@@ -1,4 +1,12 @@
 {
   "$schema": "../console-dynamic-plugin-sdk/dist/schema/console-extensions.json",
-  "data": []
+  "data": [
+    {
+      "type": "console.redux-reducer",
+      "properties": {
+        "scope": "devconsole",
```

Comment:
Maybe use `topology` instead? cc @christianvogt

Reply:
Not sure if changing now will have an impact. Best to leave as is and we can take care of it later. Topology code was moved to a separate package and further refactoring is still ongoing.